src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerHandler.cs

```diff
@@ -0,0 +1,93 @@
+        }
+
+        public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRequestParams requestParams)
+        {
+            ConnectionInfo connInfo = this.GetConnectionInfo(requestParams.ConnectionUri);
+            ServerConnection serverConnection = ConnectionService.OpenServerConnection(connInfo, ObjectManagementService.ApplicationName);
+
+            using (var context = new ServerViewContext(requestParams, serverConnection))
+            {
+                var serverSmo = CreateServerSmoObject(context).Result;
+                this.serverViewInfo.ObjectInfo = new ServerInfo()
+                {
+                    Name = serverSmo.Name,
+                    HardwareGeneration = serverSmo.HardwareGeneration,
+                    Language = serverSmo.Language,
+                    MemoryInMB = serverSmo.PhysicalMemory,
+                    OperatingSystem = serverSmo.HostDistribution,
+                    Platform = serverSmo.HostPlatform,
+                    Processors = serverSmo.Processors,
+                    IsClustered = serverSmo.IsClustered,
+                    IsHadrEnabled = serverSmo.IsHadrEnabled,
+                    IsPolyBaseInstalled = serverSmo.IsPolyBaseInstalled,
+                    IsXTPSupported = serverSmo.IsXTPSupported,
+                    Product = serverSmo.Product,
+                    ReservedStorageSizeMB = serverSmo.ReservedStorageSizeMB,
+                    RootDirectory = serverSmo.RootDirectory,
+                    ServerCollation = serverSmo.Collation,
+                    ServiceTier = serverSmo.ServiceTier,
+                    StorageSpaceUsageInGB = serverSmo.UsedStorageSizeMB,
+                    Version = serverSmo.Version.ToString(),
+                    MinServerMemory = configService.GetConfigByName(this.server, "min server memory (MB)").ConfigValue,
+                    MaxServerMemory = configService.GetConfigByName(this.server, "max server memory (MB)").ConfigValue
+            };
+
+                return Task.FromResult(new InitializeViewResult { ViewInfo = this.serverViewInfo, Context = context });
+            }
+        }
+
+        public override Task Save(ServerViewContext context, ServerInfo serverInfo)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override Task<string> Script(ServerViewContext context, ServerInfo obj)
+        {
+            throw new NotImplementedException();
+        }
+        public Task<Server> CreateServerSmoObject(ServerViewContext context)
+        {
+            using (context.Connection.SqlConnectionObject)
+            {
+                this.server = new Server(context.Connection);
+                string objectUrn = string.Format(System.Globalization.CultureInfo.InvariantCulture, "Server");
+                var serverSmo = server.GetSmoObject(new Urn(objectUrn)) as Server ?? throw new InvalidOperationException(serverNotExistsError);
```

Comment:
Would GetSmoObject already throw a FailedOperationException if the specified server doesn't exist? I've seen errors like that when getting non-existent database objects with the same method